source/fundamentals/collations.txt

```diff
@@ -0,0 +1,280 @@
+   coté
+   côte
+   côté
+
+If we apply the Canadian French collation, the order is:
+
+::
+
+   cote
+   côte
+   coté
+   côté
+
+.. note::
+
+   Collations are available in MongoDB 3.4 and later.
+
+-----
+Usage
+-----
+
+You can specify a collation when you create a **new collection** or
+**new index**. You can also specify a collation for :doc:`CRUD operations
+</fundamentals/crud>` and aggregations.
+
+When you create a **new collection** with a collation, you define the default
+collation for any of the :manual:`operations that support collation
+</reference/collation/#operations-that-support-collation>` called on that
+collection. You can override the collation for an operation by specifying a
+different one.
+
+When you create a **new index** with a collation, you specify the sort order
+for operations that use that index. To use the collation in the index, you
+must provide a matching collation in the operation, and the operation must
+use the index. While most index types support collation, the following
+types support only binary comparison:
+- :manual:`text </reference/method/db.collection.createIndex/#options-for-text-indexes>`
+- :manual:`2d </reference/method/db.collection.createIndex/#options-for-2dsphere-indexes>`
+- :manual:`geoHaystack </reference/method/db.collection.createIndex/#options-for-geohaystack-indexes>`
+
+.. note::
+
+   Currently, you cannot create a collation on an existing collection. To use
+   collations with an existing collection, create an index with the collation
+   and specify the same collation in your operations on it.
+
+Collation Parameters
+~~~~~~~~~~~~~~~~~~~~
+
+The collation object contains the following parameters:
+
+.. code-block:: javascript
+
+   collation: {
+     locale: <string>,
+     caseLevel: <bool>,
+     caseFirst: <string>,
+     strength: <int>,
+     numericOrdering: <bool>,
+     alternate: <string>,
+     maxVariable: <string>,
+     backwards: <bool>
+   }
+
+You must specify the ``locale`` field in the collation; all other fields
+are optional. For a complete list of supported locales and the default values
+for the ``locale`` fields, see :manual:`Supported Languages and Locales
+</reference/collation-locales-defaults/#supported-languages-and-locales>`.
+For descriptions of each field, see the :manual:`Collation Document MongoDB
+manual entry </reference/collation/#collation-document>`.
+
+------------------
+Collation Examples
+------------------
+
+Set a Default Collation on a Collection
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the following example, we create a new collection called ``souvenirs`` on
+the ``sample_mflix`` database and assign a default collation with the
+"``fr_CA``" locale. The collation applies to all :manual:`operations that
+support collation </reference/collation/#operations-that-support-collation>`
+performed on that collection.
+
+.. literalinclude:: /code-snippets/collation/collection-collation.js
+   :language: javascript
+   :dedent: 4
```

Comment:
It might be a good idea to add an example of a query against this collection that *does not* specify a collation, and point out that such a query uses the default collation of ``fr_CA`` that we specified above. I know when I read through this and hit the index collation example beneath I felt like there was a missing example above for the collection collation (and I like the idea of showing how to define a collation for a collection, then how to use a collation on a collection, then how to define a collation for an index, and then how to use the index based on collation)